cmd/appendGlobalCmdFlagsToConfig.go

```diff
@@ -0,0 +1,12 @@
+package cmd
+
+import (
+	"github.com/nanovms/ops/lepton"
+	"github.com/spf13/pflag"
```

Comment:
is it necessary to have another flag lib? we already have one in addition to the existing stdlib stuff

Reply:
This is an internal library used by `spf13/cobra`. We're using to receive `cmd.Flags()` result in a function.